zephyr/projects/zxa_example_project/src/main.c

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2021 Legrand North America, LLC.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(zxa);
```

Comment:
`LOG_MODULE_REGISTER(example_app)`

Reply:
See rationale above for the need for a module abbreviation.  If this should be a non-abbreviated/non-truncated name, then it should be `example_application`, no?